nautobot/users/api/views.py

```diff
@@ -42,6 +42,33 @@ def my_profile(self, request):
         serializer = self.serializer_class(instance=request.user, context={"request": request})
         return Response(serializer.data)
 
+    @action(methods=["GET"], detail=False, url_path="session", permission_classes=[AllowAny])
+    def session(self, request):
```

Comment:
Ugly as sin. Would love some thought on what else we would want in here and then I could reformat as necessary.

Reply:
`url_path` defaults to match the method name if not provided, so you could at least omit that. :)